internal/packager/init.go

```
@@ -123,6 +123,48 @@ func checkComposeFileVersion(compose map[string]interface{}) error {
 	return schema.Validate(compose, fmt.Sprintf("%v", version))
 }
 
+func getEnvFiles(svcName string, envFileEntry interface{}) []string {
+	var envFiles []string
+	switch envFileEntry.(type) {
+	case string:
+		envFiles = append(envFiles, envFileEntry.(string))
+	case []interface{}:
+		for _, env := range envFileEntry.([]interface{}) {
+			envFiles = append(envFiles, env.(string))
+		}
+	default:
+		logrus.Warnf("unknown entries in 'env_file' for service %s -> %v\n",
```

Maybe it should return an error instead of printing a warning?